### PR TITLE
Improve the documentation of the Ease module

### PR DESCRIPTION
**Pull request by noobiept**
Add missing ```@param``` and ```@return``` to the documentation of the functions in Ease.

### CODE REVIEW

File: src/tweenjs/Ease.js [hunk 1]
```diff
@@ -87,223 +92,275 @@ this.createjs = this.createjs||{};
...
-		}
-	}
+		};
+	};
 
 	/**
 	 * @method quadIn
+	 * @param {Number} t
 	 * @static
+	 * @return {Number}
```
Comment (lannymcnie) [inline src/tweenjs/Ease.js]: I think these return Function and not Number

Comment (lannymcnie) [inline src/tweenjs/Ease.js]: Sorry, my mistake, you are right.

**Comment by lannymcnie** [comment]
Thanks for the awesome PR.

**Comment by noobiept** [comment]
No problem :)

**Pull request merged**